src/ui/public/vis/editors/default/default.html

```diff
@@ -1,11 +1,11 @@
 <div class="collapsible-sidebar" ng-class="{
-    'collapsible-sidebar-small': vis.type.editorConfig.defaultSize === 'small',
-    'collapsible-sidebar-medium': vis.type.editorConfig.defaultSize === 'medium',
-    'collapsible-sidebar-large': vis.type.editorConfig.defaultSize === 'large'
+    'collapsible-sidebar--small': vis.type.editorConfig.defaultSize === 0,
```

Comment:
If we would extract this into a scope method, we could still use the constants to check:

**[in default.js]**
```js
$scope.getSidebarClass = () => {
  if ($scope.vis.type.editorConfig.defaultSize === DefaultEditorSize.SMALL) {
    return 'collapsible-sidebar--small';
  } else if (...) {
    // ...
  }
}
```

and then just apply it in here (since that cannot change, a bind once should be fine too):

```html
<div ng-class="::getSidebarClass()" ...>
```